DESCRIPTION

```diff
@@ -23,6 +23,7 @@ Biarch: true
 Depends:
     R (>= 3.3.0)
 Imports:
+    methods,
```

Comment:
Perhaps we could do the oldclass registration with a onLoad hook, so we wouldn't have to import methods? We would have to verify it works, but I think it should...

On the other hand methods is already a default package, so most people will already have it loaded anyway.

On the other other hand :) just loading methods adds a _lot_ of symbols to the symbol table, which significantly affects the time it takes to perform a gc. e.g.

```
R_DEFAULT_PACKAGES=base,compiler,graphics,utils,grDevices,stats,datasets R --quiet --vanilla -e 'system.time(for (i in 1:100) gc())'
> system.time(for (i in 1:100) gc())
   user  system elapsed
  0.447   0.002   0.449
R_DEFAULT_PACKAGES=base,compiler,graphics,utils,grDevices,stats,datasets,methods R --quiet --vanilla -e 'system.time(for (i in 1:100) gc())'
> system.time(for (i in 1:100) gc())
   user  system elapsed
  1.283   0.003   1.287
```